packages/components/psammead-media-player/src/Amp/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { Fragment } from 'react';
+import { string } from 'prop-types';
+import Helmet from 'react-helmet';
+
+const AmpHead = () => (
+  <Helmet>
+    <script
+      async
+      custom-element="amp-iframe"
+      src="https://cdn.ampproject.org/v0/amp-iframe-0.1.js"
+    />
+  </Helmet>
+);
+
+const AmpMediaPlayer = ({ src, placeholderSrc }) => {
+  return (
+    <Fragment>
+      <AmpHead />
+      <amp-iframe
+        sandbox="allow-scripts allow-same-origin"
+        layout="fill"
+        frameborder="0"
+        src={src}
+        allowfullscreen
+      >
+        <amp-img layout="fill" src={placeholderSrc} placeholder />
```

Comment:
`placeholderSrc` seems pretty vital here. Should it be `{placeholderSrc && <amp-img />}` so it doesnt render it if theres no image?

Reply:
it should be a required prop so this shouldn't be necessary. have updated proptypes.